aaa.asp

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<head>
+<title><%
+response.write("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
```

Comment:
It'd probably be unwise to keep adding on to the title, might make the supposed browser crash?